### DISCUSSION TITLE
how to get path of finetuned model ?

### DISCUSSION THREAD

**Discussion by estuday** [coqui-ai/TTS#4071] [General Q&A]
hi,
i followed the code in https://github.com/coqui-ai/TTS/blob/dev/recipes/ljspeech/xtts_v2/train_gpt_xtts.py finetuned the xttsv2. I noticed that the program will create a new folder under the passed output_path to store the fine-tuned model.
Although it will be printed in the log, but I want to get the path where the model is stored, or the path of the best model, like the two paths in the figure below, how can I do it?
store path:
![image](https://github.com/user-attachments/assets/4ac6beb4-9900-43a7-af5f-6c4942083523)
or
best model path
![image](https://github.com/user-attachments/assets/4b4a3e42-7f12-4e1e-8291-8b6c58fb00b0)

**Comment by eginhard** [comment]
Not sure what exactly you mean, you set the output path here:
https://github.com/coqui-ai/TTS/blob/dbf1a08a0d4e47fdad6172e433eeb34bc6b13b4e/recipes/ljspeech/xtts_v2/train_gpt_xtts.py#L16-L17

If you want to find the folder of the latest run, you can sort them by time. If you want to find the latest/best checkpoint programmatically, you can use:
https://github.com/idiap/coqui-ai-Trainer/blob/965321daea165107566ef39ea0eb89eb0b90930f/trainer/io.py#L254
